share/spice/github_jobs/github_jobs.js

```diff
@@ -16,24 +18,40 @@
 
         var sourceUrl = 'https://jobs.github.com/positions?description=' + encodeURIComponent(jobs['description']) + "&location=" + encodeURIComponent(jobs['location']);
 
-        Spice.add({
-            id: 'github_jobs',
-            name: 'Jobs',
-            data: api_result,
-            meta: {
-                sourceUrl: sourceUrl,
-                sourceName: 'GitHub',
-                sourceIcon: true,
-                itemType: 'Jobs'
-            },
-            templates: {
-                group: 'base',
-                options: {
-                    content: Spice.github_jobs.content
+        DDG.require('moment.js', function() {
+            Spice.add({
+                id: 'github_jobs',
+                name: 'Jobs',
+                data: api_result,
+                meta: {
+                    sourceUrl: sourceUrl,
+                    sourceName: 'GitHub',
+                    sourceIcon: true,
+                    itemType: 'Jobs'
+                },
+                normalize: function(item) {
+                    return {
+                        title: item.title,
+                        subtitle: item.company,
+                        location: item.location,
+                        description: $('<div/>').html(DDG.strip_html(item.description)).text(),
+                        url: item.url,
+                        created_at: moment(item.created_at).fromNow()
+                    }
                 },
-                detail: false,
-                item_detail: false
-            }
+                templates: {
+                    group: 'text',
+                    options: {
+                        footer: Spice.github_jobs.footer
+                    },
+                    variants: {
+                        tile: 'basic',
```

Comment:
`tileSnippet: 'large'` puts a bit more space above the footer than needed I think.